nautobot/circuits/graphql/types.py

```diff
@@ -11,3 +12,46 @@ class CircuitTerminationType(gql_optimizer.OptimizedDjangoObjectType, PathEndpoi
     class Meta:
         model = CircuitTermination
         filterset_class = CircuitTerminationFilterSet
+
+    cable_peer_circuit_termination = Field("nautobot.circuits.graphql.types.CircuitTerminationType")
+    cable_peer_front_port = Field("nautobot.dcim.graphql.types.FrontPortType")
+    cable_peer_interface = Field("nautobot.dcim.graphql.types.InterfaceType")
+    cable_peer_rear_port = Field("nautobot.dcim.graphql.types.RearPortType")
+    connected_circuit_termination = Field("nautobot.circuits.graphql.types.CircuitTerminationType")
+    connected_interface = Field("nautobot.dcim.graphql.types.InterfaceType")
+
+    def resolve_cable_peer_circuit_termination(self, args):
```

Comment:
If you move `_construct_resolver` to a non-private name and/or a graphql utilities module, you could use it for these resolvers too.